src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1142,10 +1145,16 @@ typedef struct {
     int                post_nop;
 } uct_rc_mlx5_common_clean_tx_cq_ctx_t;
 
-static int
-uct_rc_mlx5_common_iface_cq_available_total(uct_rc_mlx5_iface_common_t *iface)
+static void
+uct_rc_mlx5_common_iface_cq_available_check(uct_rc_mlx5_iface_common_t *iface)
```

Comment:
minor: maybe "inline always" to be sure that compiler adds nothing in release? (but good compiler should do the same for just `static`)

Reply:
IMO it's not that important to fix for int3 branch cleanup flow

Reply:
ok